docs/SETUP.md

```diff
@@ -185,3 +185,23 @@ code ~/esp/Super-2024-Swadge-FW
 ## Updating ESP-IDF
 
 On occasion the ESP-IDF version used to build this project will increment. The easiest way to update ESP-IDF is to delete the existing one, by default installed at `~/esp/esp-idf/`, and the tools, by default installed at `~/.espressif/`, and follow the guide above to clone the new ESP-IDF and run the install script.
+
+Alternatively, you can update the IDF in-place with the following commands. This example updates the IDF to 5.3.1, and you can change that version as is necessary. These are Linux commands, so they may need to be tweaked slightly for Windows.
+
+```bash
+# Change directory to where the IDF is installed
+cd ~/esp/esp-idf/
+
+# Update the IDF with git
+git fetch --prune
+git checkout tags/v5.3.1
+git pull
```

Comment:
Is `git pull` really necessary here? This is what I get when I run `git pull` after checking out the tag:

```
You are not currently on a branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>
```

Plus I think `git fetch` does everything `git pull` does, other than updating the local branch

Reply:
Yeah, definitely added that out of git-paranoia. I'll fix that. The nice thing here is that the FW will still compile against 5.2.1 cleanly. The one breaking change was accounted for :)